src/content/1.7/development/webservice/tutorials/change_product_image.md

```diff
@@ -9,7 +9,7 @@ To create a new image we are going to use the `/images/products` API. We won't u
 
 ```php
 <?php
-    $url = 'http://example.com';
+    $url = 'https://example.com';
```

Comment:
As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?